api/src/main/java/org/openmrs/Privilege.java

```diff
@@ -9,21 +9,29 @@
  */
 package org.openmrs;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
 /**
  * Privilege
  * 
  * @version 1.0
  */
+@Entity
+@Table(name = "privilege")
+@AttributeOverride(name = "description", column = @Column(name = "description", length = 250))
 public class Privilege extends BaseChangeableOpenmrsMetadata {
 	
 	public static final long serialVersionUID = 312L;
 	
-	// Fields
-	
+	@Id
+	@Column(name = "privilege",nullable = false,length = 250)
 	private String privilege;
```

Comment:
apart from privilege, do we have other attributes defined as  required!!

Reply:
Nop, its just privilege,description and uuid